scripts/bash/getInternalDependencies.sh

```diff
@@ -1,11 +1,13 @@
-#!/bin/bash
+#!/bin/bash -e
 
 DIR=$(dirname "$0")
-package_path=$1
+if [ "$1" != "" ]; then
+  # pop the package_path off, and interpret the rest as dependencies that have been checked earlier
+  # in the recursion
+  package_path=$1
+  shift
+fi
 
-# pop the package_path off, and interpret the rest as dependencies that have been checked earlier
-# in the recursion
-shift
```

Comment:
This was actually throwing up a `1` exit code if no `package_path` was provided, so now we do it conditionally